go.mod

```diff
@@ -24,4 +25,8 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 
 replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
-replace github.com/tendermint/tendermint => github.com/forbole/tendermint v0.34.13-0.20210820072129-a2a4af55563d
+replace github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v0.34.14-celestia.0.20220119223021-e54097114e14
+
+replace github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v0.44.1-celestia
+
+replace github.com/forbole/juno/v2 v2.0.0-20211221122008-f95aacf17add => github.com/celestiaorg/juno/v2 v2.1.0-celestia
```

Comment:
super minor optional nit: we might want to format these in a more normal way similar to https://github.com/celestiaorg/celestia-app/blob/2015a0cac19c841d5971c1155c53fa2b9f736d6b/go.mod#L125-L131

Reply:
done, thanks